platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -352,6 +352,8 @@ protected void onGodotSetupCompleted() {
 		if (godotHost != null) {
 			godotHost.onGodotSetupCompleted();
 		}
+		int rotaryInputAxis = java.lang.Integer.parseInt(GodotLib.getGlobal("input_devices/pointing/android/rotary_input_scroll_axis"));
+		GodotInputHandler.setRotaryInputAxis(rotaryInputAxis);
```

Comment:
Can you move this logic within the `runOnUiThread(...` block

Reply:
fixed